.github/continuous-integration.yml

```diff
@@ -0,0 +1,40 @@
+# https://help.github.com/en/categories/automating-your-workflow-with-github-actions
+
+name: "Continuous Integration"
+
+on:
+  pull_request:
+  push:
+    branches:
+      - "master"
```

Comment:
This will run a build for pushes into `master` (e.g., when a pull request is merged or a commit is pushed directly into `master`).